packages/headless/src/utils/url-utils.ts

```diff
@@ -0,0 +1,35 @@
+export class URLPath {
```

Comment:
Lol I was 100% with you but @btaillon doesn't want me to use URL because maybe react-native has some different behaviour. That makes sense too but I was asked to come up with a util so here it is. We're not going back now ;)